### PR TITLE
Dev 1

### PR DESCRIPTION
**Pull request by DavidMikutishvili**

### CODE REVIEW

File: Deployment/Scripts/Bookstore_init.sql [hunk 1]
```diff
@@ -1,37 +1,37 @@
-CREATE DATABASE Bookstore;
-GO
-
-USE Bookstore;
-
-CREATE TABLE Authors
-(
```
Comment (Coolerbkv) [inline Deployment/Scripts/Bookstore_init.sql]: please formatting like as 
https://www.w3schools.com/sql/sql_create_table.asp

File: Deployment/Scripts/Bookstore_init.sql [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-	id INT PRIMARY KEY IDENTITY (1, 1) NOT NULL,
-	author_name NVARCHAR(500)
-);
-
-CREATE TABLE Publishers
-(
```
Comment (Coolerbkv) [inline Deployment/Scripts/Bookstore_init.sql]: please formatting like as
https://www.w3schools.com/sql/sql_create_table.asp

File: Deployment/Scripts/Bookstore_init.sql [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
-	id INT PRIMARY KEY IDENTITY (1, 1) NOT NULL,
-	publisher_name NVARCHAR(500)
-);
-
-CREATE TABLE Languages
-(
```
Comment (Coolerbkv) [inline Deployment/Scripts/Bookstore_init.sql]: please formatting like as
https://www.w3schools.com/sql/sql_create_table.asp

File: Deployment/Scripts/Bookstore_init.sql [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
+);
+
+CREATE TABLE Languages
+(
+	id INT PRIMARY KEY IDENTITY (1, 1) NOT NULL,
+	language_name NVARCHAR(500)
+);
+
+CREATE TABLE Books
+(
```
Comment (Coolerbkv) [inline Deployment/Scripts/Bookstore_init.sql]: please formatting like as
https://www.w3schools.com/sql/sql_create_table.asp

File: Deployment/Scripts/Bookstore_init.sql [hunk 5]
```diff
@@ -1,37 +1,37 @@
...
-	product_rating INT,
-	price INT,
-	author_id INT,
-	publisher_id INT,
-	language_id INT,
-	FOREIGN KEY (author_id) REFERENCES Authors (id),
-	FOREIGN KEY (publisher_id) REFERENCES Publishers (id),
-	FOREIGN KEY (language_id) REFERENCES Languages (id)
-);
+﻿CREATE DATABASE Bookstore;
```
Comment (Coolerbkv) [inline Deployment/Scripts/Bookstore_init.sql]: I propose rename script like as `Bookstore_database_init.sql`

File: Deployment/Scripts/Bookstore_init.sql (hunk 1 above)
Comment (DavidMikutishvili) [reply Deployment/Scripts/Bookstore_init.sql]: it was like this

CREATE TABLE Authors
(
    id INT PRIMARY KEY IDENTITY (1, 1) NOT NULL,
    author_name NVARCHAR(500)
); 

and I need to do it like this

CREATE TABLE Authors (
    Id int primary key identity (1, 1) not null,
    AuthorName nvarchar(500)
);

File: Deployment/Scripts/Bookstore_init.sql [hunk 6]
```diff
@@ -1,28 +1,24 @@
-﻿CREATE DATABASE Bookstore;
+CREATE DATABASE Bookstore;
 GO
 
 USE Bookstore;
 
-CREATE TABLE Authors
-(
+CREATE TABLE Authors(
```
Comment (Coolerbkv) [inline Deployment/Scripts/Bookstore_init.sql]: please use space between `Authors` and `(` like this `CREATE TABLE Authors (`

File: Deployment/Scripts/Bookstore_init.sql (hunk 1 above)
Comment (SiarheiHasiuk) [reply Deployment/Scripts/Bookstore_init.sql]: @DavidMikutishvili  Could you please commit suggested  by @Coolerbkv changes ?

Comment (DavidMikutishvili) [reply Deployment/Scripts/Bookstore_init.sql]: #https://github.com/DavidMikutishvili/Bookstore/tree/DEV-1/Deployment/Scripts

**Review by Coolerbkv — APPROVED** [review]

**Review by SiarheiHasiuk — APPROVED** [review]

**Pull request merged**